<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeEditWidget</class>
 <widget class="QWidget" name="TimeEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>26</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QAbstractSpinBox
{
    min-height: 0px;
    border: 0px solid #B9B9B9;
    color: #4A4A4A;
    background-color: #E1E1E1;
    padding-left: 20px;
}
QAbstractSpinBox:hover
{
    border: 0px solid #00A2E8;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="MySpinBox" name="spinBox_beginHour">
     <property name="maximum">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="style" stdset="0">
      <string>normal</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="MySpinBox" name="spinBox_beginMinute">
     <property name="maximum">
      <number>59</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>--</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="style" stdset="0">
      <string>normal</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="MySpinBox" name="spinBox_endHour">
     <property name="maximum">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="style" stdset="0">
      <string>normal</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="MySpinBox" name="spinBox_endMinute">
     <property name="maximum">
      <number>59</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MySpinBox</class>
   <extends>QSpinBox</extends>
   <header>myspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
